cmd/status.go

```diff
@@ -165,6 +165,8 @@ func getRole(pid int) (role string, err error) {
 		return "controller", nil
 	} else if strings.Contains(cmdln, "worker") {
 		return "worker", nil
+	} else if strings.Contains(cmdln, "server") {
```

Comment:
Perhaps adding OR to the line 164 would make it nicer
```
} else if strings.Contains(cmdln, "controller") || else if strings.Contains(cmdln, "server") {
...
```

Reply:
IMO this is better as separate branch as we're not going to keep it for long. Makes it stand out better so we remember to clean it after some time.